### PR TITLE
✅ [amp-base-carousel] Use version property in spec and `-latest.js` in test fixtures

### PR DESCRIPTION
**Pull request by rcebulko**

### CODE REVIEW

File: extensions/amp-base-carousel/0.1/test-e2e/test-advance.js [hunk 1]
```diff
@@ -20,8 +20,9 @@ const pageWidth = 500;
 const pageHeight = 800;
 
 describes.endtoend(
-  'amp-base-carousel:0.1 - advance',
+  'amp-base-carousel - advance',
   {
+    version: '0.1',
```
Comment (caroqliu) [inline extensions/amp-base-carousel/0.1/test-e2e/test-advance.js:25]: To confirm, does the console output for failed tests include the component version? The original reason they were added to the test names was so we could identify which version to debug.

File: test/fixtures/e2e/amp-base-carousel/zero-index-slide.amp.html [hunk 2]
```diff
@@ -6,18 +6,18 @@
...
   </head>
 
   <body>
     <amp-base-carousel
         id="carousel"
-      auto-advance="true" 
-        auto-advance-interval="3000" 
-        height="120" 
-        snap-align="center" 
+      auto-advance="true"
```
Comment (caroqliu) [inline test/fixtures/e2e/amp-base-carousel/zero-index-slide.amp.html:15]:
```suggestion
        auto-advance="true"
```

File: extensions/amp-base-carousel/0.1/test-e2e/test-advance.js (hunk 1 above)
Comment (rcebulko) [reply extensions/amp-base-carousel/0.1/test-e2e/test-advance.js:25]: Yes; `describes.endtoend` embeds the version # inside the test name when the param is present:
image

Comment (caroqliu) [reply extensions/amp-base-carousel/0.1/test-e2e/test-advance.js:25]: Cool, thanks!

**Review by caroqliu — APPROVED** [review]

**Pull request merged**